### ISSUE TITLE
Update Popover UX guidelines to avoid time-based auto-dismissal

### ISSUE THREAD

**Issue by Blackbaud-AlexKingman** [blackbaud/skyux-popovers#92]
We added a programmatic option to dismiss the popover in `2.26.0`, but did not guidance on how to use this while taking accessibility concerns in mind.

See issue #13
